src/WebJobs.Script/Description/Worker/WorkerLanguageInvoker.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.Azure.WebJobs.Script.Binding;
+using Microsoft.Azure.WebJobs.Script.Description.Script;
+using Microsoft.CodeAnalysis;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    // TODO: make this internal
+    public class WorkerLanguageInvoker : FunctionInvokerBase
```

Comment:
Wonder if we shouldn't rename this to RpcFunctionInvoker? Kind of matches your WebJobs.Script.Rpc nomenclature.

Reply:
Sounds good. Worker is a super overloaded term, so RPC is perhaps a bit more descriptive.

Reply:
I agree that it would be nice to rename this. I really don't like to `Rpc` name all over. We've talked about that, but never came up with an alternative.